YOLO.xcodeproj/project.pbxproj

```diff
@@ -351,7 +331,7 @@
 				ASSETCATALOG_COMPILER_INCLUDE_ALL_APPICON_ASSETS = NO;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 0;
-				DEVELOPMENT_TEAM = 3MR4P6CL3X;
+				DEVELOPMENT_TEAM = "";
```

Comment:
Reading around there is no security issue in keeping DEVELOPMENT_TEAM public, let's leave it like this:
```suggestion
				DEVELOPMENT_TEAM = 3MR4P6CL3X;
```

Reply:
I reverted DEVELOPMENT_TEAM to original.